src/postcss/atomic-css-modules/generateScopedName.ts

```diff
@@ -58,20 +57,6 @@ const generateScopedName: generateScopedName = ({
         } else {
           pkgName = importedPackage;
         }
-      } else if (/:import/.test(rule.selector)) {
```

Comment:
@sylvesteraswin this seem to be the culprit: what was happening basically is that classes in a file with a css import were being evaluated twice with one f the time getting the package name of the imported module (e.g. @my-org/simple-class;withSharedVar).

removing it and returning when rule is a ISS import seems to work fine with the hashes and I also ran the integration tests multiple times to check consstency